### PR TITLE
Added sharedAction example to Share API

### PR DESCRIPTION
**Pull request by kshitij86**
## Summary
* Added an example for `sharedAction` and `dismissedAction` in Share API examples

**Motivation** - Missing example for `sharedAction` in examples

## Changelog

[General] [Added] - Added an example in ShareExample.js

## Test Plan
* Tested on Android

sharedAction

### CODE REVIEW

**Comment by mkcode** [comment]
This great to me.

**Comment by kshitij86** [comment]
@rickhanlonii Could you please review these changes?

**Comment by rickhanlonii** [comment]
Looks like the android test failed, I re-ran the jobs and if it passes I'll import it.

File: Libraries/Components/ScrollView/ScrollViewStickyHeader.js [hunk 1]
```diff
@@ -177,7 +177,7 @@ class ScrollViewStickyHeader extends React.Component<Props, State> {
     // Fabric Detection
     // eslint-disable-next-line dot-notation
     const isFabric = !!(
-      this._ref && this._ref['_internalInstanceHandle']?.stateNode?.canonical
+      this._ref && this._ref._internalInstanceHandle?.stateNode?.canonical
```
Comment (rickhanlonii) [inline Libraries/Components/ScrollView/ScrollViewStickyHeader.js:180]: Could we revert this?

**Pull request closed without merging**